pkg/approver/manager/internal_test.go

```diff
@@ -0,0 +1,251 @@
+/*
+Copyright 2021 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package manager
+
+import (
+	"context"
+	"errors"
+	"path/filepath"
+	"testing"
+
+	cmapi "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
+	cmmeta "github.com/jetstack/cert-manager/pkg/apis/meta/v1"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	cmpapi "github.com/cert-manager/policy-approver/pkg/apis/policy/v1alpha1"
+	"github.com/cert-manager/policy-approver/pkg/approver"
+	"github.com/cert-manager/policy-approver/pkg/approver/fake"
+	"github.com/cert-manager/policy-approver/pkg/approver/manager/predicate"
+	"github.com/cert-manager/policy-approver/test/env"
+)
+
+func Test_Review(t *testing.T) {
+	rootDir := env.RootDirOrSkip(t)
```

Comment:
> Maybe the test should just fail here if ROOTDIR not defined?

@irbekrm I agree, Go is not great in this area with not doing something like aggregating results like ginkgo does. I do want to have this skip though since I am running `go test` in my editor and terminal for running the non-API server dependent tests for various packages. FWIW, the `Makefile` does do testing verbose, and I personally have a terminal alias `got` when running tests. I'll leave it as is for now.

> Is there a need to support running from many rootDirs?

No, it needs to have the root dir so it can build paths to the dependencies, and have the soft dependency of the test execution coming from the Makefile since it relies on the `ROOTDIR` environment variable being defined. I don't know of a reliable way of getting the Go project root using the stdlib.